### DISCUSSION TITLE
Direct Messages?

### DISCUSSION THREAD

**Discussion by itsWindows11** [bluesky-social/atproto#1728] [General Discussion]
Bluesky (or any federated platform as far as I know) currently doesn't have direct messaging. It might be good to add direct messaging, and replicate Twitter's functionality in terms of direct message privacy too.

Something that could also be looked into is direct message groups, but it doesn't have to be initially there with DMs.

**Comment by snarfed** [comment]
https://github.com/bluesky-social/atproto/discussions/121
